include/lsst/afw/table/Schema.h

```diff
@@ -272,6 +272,17 @@ class Schema {
     /// @brief Copy constructor.
     Schema(Schema const & other);
 
+    /// @brief Construct from reading a FITS file.
+    ///
+    /// Reads from the nominated 'hdu' (1=PHU which cannot be a catalog).
+    static Schema readFits(std::string const& filename, int hdu=2);
```

Comment:
Do we want to add the `Fits` and `MemFileManager` overloads too?  I'm not sure they're useful here, but that's the established pattern.